app/reconciler/readRuote53IntoCompareStructure.server.ts

```diff
@@ -0,0 +1,70 @@
+import { set } from 'lodash';
+import { getRecordPage } from '~/lib/dns.server';
+
+// Using this in JS code later, cannot `import type`
+import { DnsRecordType } from '@prisma/client';
+import type { ReconcilerCompareStructure } from './ReconcilerTypes';
+import type { ResourceRecordSet, ListResourceRecordSetsResponse } from '@aws-sdk/client-route-53';
+
+/**
+ * !!! There is cross-function object mutation in this file.
+ * This is a huge anti-pattern, but in this case it made the code
+ * much more performant. So I marked the variable as such and
```

Comment:
Your description for this issue should include details about this.  Do we care if it's slow, given that this is a background job?  What were your findings?

I think the way that you've marked it in both places is good.